WordPress/openverse
# Update close buttons

## Fixes



Fixes #467 by @obulat

## Description


This PR replaces `VCloseButton` with `VIconButton` for the close button in several components. This uses the updated button styles and removes the old button styles from the app.

### The license explanation popup
On desktop widths
<img width="312" alt="Screenshot 2023-10-19 at 10 51 38 AM" src="https://github.com/WordPress/openverse/assets/15233243/d39e9f7e-404e-4adc-aeca-4849898c5c9f">

### The report popup
Single result pages, both desktop and mobile widths
<img width="337" alt="Screenshot 2023-10-19 at 11 21 23 AM" src="https://github.com/WordPress/openverse/assets/15233243/87703de3-38a6-4cab-b13f-3392986b9889">

### The pages modal
Content pages, i.e. `/about`, on mobile widths
<img width="518" alt="Screenshot 2023-10-19 at 11 22 07 AM" src="https://github.com/WordPress/openverse/assets/15233243/769a5ace-d3dc-41b2-b719-a35bd5e6dcbf">

### The content settings modal
Search page on mobile widths
<img width="520" alt="Screenshot 2023-10-19 at 11 23 05 AM" src="https://github.com/WordPress/openverse/assets/15233243/a66254de-18c7-4f95-86e6-3c0c2739e157">

### Notification bar
Dark mode on homepage, light mode on the search page
<img width="522" alt="Screenshot 2023-10-19 at 11 23 35 AM" src="https://github.com/WordPress/openverse/assets/15233243/0448ebf8-5da5-4c5e-9628-8e6c0fc6b8a2">

<img width="523" alt="Screenshot 2023-10-19 at 11 23 17 AM" src="https://github.com/WordPress/openverse/assets/15233243/e79d1403-7d1c-45d1-ac64-708c3e8ce027">

### Modal content
External sources form, images search page
<img width="484" alt="Screenshot 2023-10-19 at 11 31 11 AM" src="https://github.com/WordPress/openverse/assets/15233243/1f46445f-dba6-4cb1-8cec-cb7cfa91a62d">


This is a part of the effort of replacing `VOldIconButton` that is used by `VCloseButton`. Now, the only component that uses `VCloseButton` is the global audio player, which will be updated in a separate PR since the design was changed recently.

## Testing Instructions

Run the app and open the search page (localhost:8443/search/?q=cat) on a desktop width. 

For the license explanation button, open the filters sidebar and click on a license explanation button `(?)`. Check that the close button's focus ring looks correct.
Check other buttons listed in the description of the PR.

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
